.github/workflows/build.yml

```diff
@@ -18,4 +18,4 @@ jobs:
     - name: Use current action
       uses: ./
       with:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Confirmed this is the correct input: https://github.com/getsentry/action-github-commit/blob/main/action.yml#L8